app/navigation/Navigator/AppNavigator.tsx

```diff
@@ -3,10 +3,10 @@ import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
 
 import ModalNavigator from "./ModalNavigator";
-import MethodologyModalNavigator from "./MethodologyModalNavigator";
+import HtmlViewModalNavigator from "./HtmlViewModalNavigator";
```

Comment:
Let's be closer to reality in the naming : let's use `InfoModalNavigator` instead of `HtmlViewModalNavigator` is better I think

Reply:
Agreed, changing it over!